src/Voron/Impl/Paging/AbstractPager.cs

```diff
@@ -185,6 +189,17 @@ public PagerState EnsureContinuous(long requestedPageNumber, int numberOfPages)
             return AllocateMorePages(allocationSize);
         }
 
+        [Conditional("VALIDATE")]
```

Comment:
Yes. 17.4.2.1 of the C# spec says: "The conditional method must not be marked with the override modifier. A conditional method may be marked with the virtual modifier, however. Overrides of such a method are implicitly conditional, and must not be explicitly marked with a Conditional attribute."